src/olympia/amo/tests/ui/conftest.py

```diff
@@ -0,0 +1,64 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import datetime
+import os
+import urlparse
+
+from fxapom.fxapom import DEV_URL, PROD_URL, FxATestAccount
+import jwt
+import pytest
+import requests
+
+
+@pytest.fixture
+def fxa_account(base_url):
+    url = DEV_URL if 'dev' in base_url else PROD_URL
+    return FxATestAccount(url)
```

Comment:
This suite can be used to run against any deployed instance of AMO. This is controlled by passing `--base-url` on the command line.